au4/src/effects/effects_base/qml/Audacity/Effects/EffectsViewerDialog.qml

```diff
@@ -58,5 +49,27 @@ StyledDialogView {
         Component.onCompleted: {
             bbox.buttonById(ButtonBoxModel.Apply).enabled = false
         }
+
```

Comment:
There was a request from Product to have the `Apply` button being the default one and with blue highlight, like in the save-on-close dialog. Would you please add this?